node/out_request.js

```diff
@@ -104,6 +104,14 @@ TChannelOutRequest.prototype._sendCallRequestCont = function _sendCallRequestCon
     });
 };
 
+TChannelOutRequest.prototype._sendPingRequest = function _sendPingRequest() {
```

Comment:
this should not be on the req res objects, it's a concern external to call req/res.